docs/source/bali/index.rst

```diff
@@ -0,0 +1,74 @@
+====
+Bali
+====
+
+Bali is the planned third release of the OpenSDS project. It is currently under
+development with a planned release date of the week of December 10, 2018.
+
+Bali Schedule
+=============
+
+.. datatemplate::
+   :source: schedule.yaml
+   :template: schedule_table.tmpl
+
+.. note::
+
+   These dates are based on the current plan and are subject to change.
+
+Bali Milestones
+===============
+
+.. _b-1:
+
+Milestone 1
+-----------
+
+The first Bali milestone will be the week of August 6.
+
+This first milestone serves as a checkpoint on the progress so far and as a
+good point to make sure our release processes are in order.
+
+.. _b-2:
+
+Milestone 2
+-----------
+
+The second Bali milestone will be the week of September 24.
+
+No new features may be proposed for the Bali release without the review and
+approval of the Technical Steering Committee (TSC).
```

Comment:
Do you mean no new features may be proposed without the review/approval after milestone 2?

Reply:
Correct. I think after this point we would need to have a discussion at the TSC level as to whether any planned feature changes would be of high enough priority to change plans. After milestone-2 we would want to focus just on getting our currently planned features fully implemented and tested.

Reply:
Can you change to "No new features may be proposed for the Bali release without the review and approval of the Technical Steering Committee (TSC) after the second milestone" to make it clear?

Reply:
This is under the second milestone section, so that's pretty much exactly what it currently says.

Maybe looking at the generated output would make this more clear?

Reply:
Under Milestone 3, you specified that "All new feature development should be complete by the third milestone."  I think it is more clear to specify milestone-2 here.

Reply:
Milestone 2 would be way too early for that I think. Would that be possible? Then what would happen between m-2 and m-3?

Reply:
I didn't mean to change the text for "All new features...".  What I was trying to say is that I like the sentence under Milestone 3 where it clearly stated what needs to be done by m-3.  I'd like to see "after the second milestone" added in this section for "No new features may be proposed" so it is more clear.  I hope I'm not adding more confusion to my comments here:).